src/functions-and-arrays.js

```diff
@@ -52,16 +82,25 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+function uniquifyArray(wordsUniqueArray) {
 
+}
```

Comment:
Check the lab solution for this one and feel free to reach out if you have any questions.